src/Essentials/test/DeviceTests/Tests/FileSystem_Tests.cs

```diff
@@ -57,5 +57,19 @@ public async Task ValidateMIMEFormat()
 			File.Delete(filePath);
 		}
 #endif
+		[Fact]
```

Comment:
Could add a test that calls `OpenReadAsync` twice to validate it? Ensures the second call does not regress and that the StorageFile remains usable across multiple streams.